src/Spice86.Core/Emulator/InterruptHandlers/VGA/Records/VbeInfoBlock.cs

```diff
@@ -0,0 +1,116 @@
+namespace Spice86.Core.Emulator.InterruptHandlers.VGA.Records;
+
+using Spice86.Core.Emulator.InterruptHandlers.VGA.Enums;
+using Spice86.Core.Emulator.Memory.Indexable;
+using Spice86.Shared.Emulator.Memory;
+using Spice86.Shared.Utils;
+
+/// <summary>
+/// VBE Controller Information Block (256 bytes).
+/// This structure contains information about the VBE controller and available video modes.
+/// Programs typically call Function 00h (ReturnControllerInfo) first to detect VBE presence.
+/// </summary>
+public class VbeInfoBlock {
```

Comment:
This should inherit from MemoryBasedDataStructure, and follow the pattern of putting known constants in the MemoryMap.cs file.